priv/site/_includes/css/components/Schedule.scss

```diff
@@ -0,0 +1,209 @@
+
+.Schedule-hero {
+  display: flex;
+  flex-direction: column;
+}
+
+.Schedule-heroImage {
+  display: none;
+  width: 50%;
+}
+
+.Schedule-content {
+  padding: var(--s-10) var(--s-4) 0;
+}
+
+.Schedule-contentDateTime {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-bottom: var(--s-6);
+  color: var(--c-black);
+  font-size: var(--s-4);
+  font-weight: 600;
+  line-height: 28px;
+  letter-spacing: 0.02em;
+  text-transform: uppercase;
+}
+      
+.Schedule-contentTitle {
+  padding: var(--s-3);
+  padding-right: var(--s-6);
+  margin-bottom: var(--s-6);
+  background-color: var(--c-gray);
+  color: var(--c-black);
+  font-size: var(--s-7);
+  line-height: 90%;
+  font-weight: normal;
+  letter-spacing: -0.02em;
+  text-transform: uppercase;
+}
+
+.Schedule-list li {
+  display: flex;
+  justify-content: space-between;
+  padding: var(--s-2) 0;
+  border-top: 1px solid var(--c-black);
+  color: var(--c-black);
+}
+
+.Schedule-list li:last-child {
+  border-bottom: 1px solid var(--c-black);
+}
+
+.Schedule-listTime {
+  font-family: "Chakra Petch";
+  font-weight: 600;
+}
+
+.Schedule-listTalkTitle {
+  font-family: "input-mono";
+  font-weight: 400;
+}
+
+.Schedule-listSpeaker {
+  display: none;
+}
+
+.Schedule .marquee {
+  background-color: var(--c-light-gray);
+  color: var(--c-black);
+  font-size: var(--s-8);
+  line-height: var(--s-9);
+  text-transform: uppercase;
+}
+
+.Speakers {
+  background-color: var(--c-gray);
+}
+
+.Speakers-speakerPhoto {
+  padding-top: var(--s-5);
+  padding-bottom: var(--s-5);
+  display: grid;
+  justify-content: center;
+}
+
+.Speakers-speakerPhoto img, .Speakers-speakerName {
+  border: 1px solid var(--c-black);
+}
+
+.Speakers-speakerName {
+  background-color: var(--c-white);
+  text-align: center;
+  color: var(--c-black);
+  font-size: var(--s-5);
+  line-height: var(--s-7);
+  text-transform: uppercase;
+  font-family: "Chakra Petch";
+  font-weight: 500;
+  border-top: 0;
+}
+
+.Speakers-contentTalkTitle {
+  text-align: center;
+  color: var(--c-black);
+  font-size: var(--s-4);
+  line-height: var(--s-5);
+  text-transform: uppercase;
+  font-family: "Chakra Petch";
+  font-weight: 500;
+  padding: 0 var(--s-6) var(--s-2);
+}
+
+@media (max-width: 650px) {
+  .Schedule-content {
+    padding: var(--s-5);
+  }
+}
+
+@media (min-width: 1200px) {
+
+  .u-desktop {
+    display: block;
+  }
+
+  .Schedule-hero {
+    flex-direction: row-reverse;
+  }
+
+  .Schedule-heroImage {
+    display: inherit;
+    max-height: 1258px;
+  }
+
+  .Schedule-content {
+    width: 50%;
+    padding: var(--s-10) var(--s-7) var(--s-8);
+  }    
+
+  .Schedule-contentTitle {
+    font-size: var(--s-9);
+  }
+
+  .Schedule-listSpeaker {
+    display: inherit;
+  }
+
+  .Schedule-list li:last-child {
+    border-bottom: 1px solid var(--c-black);
+  }
+
+  .Schedule-list li:nth-child(odd) {
+    > span:nth-child(2) {
+      color: var(--c-white);
+    }
+  }
+
+  .Speakers-content {
```

Comment:
This is confusing, if we´re resusing any styles from speakers and/or adding to them we should move these rules to `Speakers.scss`. If we aren't, which seems to be the case, could we name this based on `Schedule`?